howto/manage-relations.md

```diff
@@ -46,7 +46,7 @@ jimmctl auth relation list [options]
 Given two entities A and B and a pre-existing relation between them, to remove the relation, run the `auth relation remove` command followed by the tag of A, the relation, and the tag of B. For example:
 
 ```text
-jimmctl auth relation remove user-alice@canonical.com member group-mygroup
+juju remove-permission user-alice@canonical.com member group-mygroup
 ```
 
-> > See more: {ref}`view-all-the-current-relations`, {ref}`jimmctl auth relation remove <summary-13>`
\ No newline at end of file
+> > See more: {ref}`view-all-the-current-relations`, {ref}`juju remove-permission <summary-13>`
```

Comment:
Here and in other places: The anchor is wrong. (Clicking on `juju remove-permission` shows `jaas import-model`.)

Reply:
But `<summary-13>` was not the start of the reference doc. Anyway, I can fix that later myself.

Reply:
Yip it wasn't, but if we take the time to manually link to each `summary-X` right now, if we add/remove a command from/to the jaas plugin, all the numbers will change and the links will point to the wrong commands without anyone noticing. So I think it's better to get the auto-generated docs to have more concrete reference tags.